docs/samples/deployments/spark/presidio_anonymize_blobs.py

```diff
@@ -88,7 +88,7 @@ def anonymize_text(text: str) -> str:
     anonymized_results = anonymizer.anonymize(
         text=text,
         analyzer_results=analyzer_results,
-        anonymizers_config={
+        operators={
```

Comment:
this change means another minor version. @omri374 are we good with that?

Reply:
@balteravishay, @omri374 was the requestor of this change :)